### PR TITLE
Repair `eventgrid` ci job

### PR DESCRIPTION
**Pull request by scbedd**
See title.

### CODE REVIEW

File: sdk/eventgrid/ci.yml [hunk 1]
```diff
@@ -44,6 +44,11 @@ parameters:
   displayName: 'azure-messaging-eventgrid-cloudnative-cloudevents'
   type: boolean
   default: true
+- name: release_azureresourcemanagereventgrid
```
Comment (weshaggard) [inline sdk/eventgrid/ci.yml:47]: I think this removal was intentional as it was moved to its own pipeline https://github.com/Azure/azure-sdk-for-java/blob/54c199dde1ef2bf738bcc838cdb6b87b9d2cf9c6/sdk/eventgrid/azure-resourcemanager-eventgrid/ci.yml

File: sdk/eventgrid/ci.yml [hunk 2]
```diff
@@ -45,6 +45,7 @@ parameters:
   type: boolean
   default: true
 
+
```
Comment (weshaggard) [inline sdk/eventgrid/ci.yml:48]:
```suggestion
```

**Review by weshaggard — APPROVED** [review]

**Comment by scbedd** [comment]
Failure not related to this PR. Overriding.

**Pull request merged**